recipe/0219-using-caption-file/manifest.json

```diff
@@ -54,7 +54,7 @@
                     "Captions in WebVTT format"
                   ]
                 },
-                "language": "en"
+                "language": ["en"]
```

Comment:
Because the annotation model (and the IIIF Specification itself) has single languages as string values in annotation bodies - I think this should too.